utils/storybook/Markdown.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import 'github-markdown-css';
+import EmojiConvertor from 'emoji-js';
+import MarkdownToJSX from 'markdown-to-jsx';
+
+const emoji = new EmojiConvertor();
+
+interface MarkdownProps {
+  content: string;
+}
+
+export default class Markdown extends React.Component<MarkdownProps> {
```

Comment:
No need to refactor this, but in the future, let's use function components for this - it's just way simpler:
```jsx
export default function(props: { content: string }) {
  const ConvertedContent = emoji.replace_colons(props.content);
  return <MarkdownToJSX className="markdown-body">{ConvertedContent}</MarkdownToJSX>;
}
```

Reply:
Good to know. I was wondering if there was a better way to write this 😅

Reply:
Agreed

Reply:
Although probably better to name the function, otherwise dev tools show this component as anonymous